SCRAM/GMake/Makefile.rules

```diff
@@ -2064,15 +2062,15 @@ ifneq ($(strip $(EDM_CHECK_CLASS_TRANSIENTS)),)
 ifeq ($(strip $(NO_CAPABILITIES)),)
 define check_edm_class_transients
   && $(CMD_echo) ">> Checking EDM Class Transients in $(@F)" &&\
-  $(VERB_ECHO) $(OS_RUNTIME_LIBRARY_PATH)=$$LOCALTOP/$(@D):$$$(OS_RUNTIME_LIBRARY_PATH) $(EDM_CHECK_CLASS_TRANSIENTS_SCRIPT) -l $(@F) $(foreach r,$2,-f $r) &&\
-  (($(OS_RUNTIME_LIBRARY_PATH)=$$LOCALTOP/$(@D):$$$(OS_RUNTIME_LIBRARY_PATH) $(EDM_CHECK_CLASS_TRANSIENTS_SCRIPT) -l $(@F) $(foreach r,$2,-f $r)) || ($(call delete_plugin_build_prod,$(3),$(4)) && exit 1)) &&\
+  $(VERB_ECHO) cd $(LOCALTOP)/$(@D)/a \&\& $(EDM_CHECK_CLASS_TRANSIENTS_SCRIPT) -l $(@F) $(foreach r,$2,-f $r) &&\
+  (($(CMD_cd) $(LOCALTOP)/$(@D)/a && $(EDM_CHECK_CLASS_TRANSIENTS_SCRIPT) -l ../$(@F) $(foreach r,$2,-f $(LOCALTOP)/$r)) ) &&\
```

Comment:
do not understand why you need this? Library should already been copied to localtop/lib/arch directory?